images/iqsharp-base/Dockerfile

```diff
@@ -50,6 +51,7 @@ RUN wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor
     chown root:root /etc/apt/sources.list.d/microsoft-prod.list && \
     apt-get -y update && \
     apt-get -y install dotnet-sdk-3.1 && \
+    apt-get -y install dotnet-sdk-6.0 && \
```

Comment:
Very minor, but it may be faster to build this image with these grouped in

Reply:
Makes sense. :)